src/js/views/dashboards/dashboard_views.js

```diff
@@ -26,7 +26,7 @@ const ContextTrailView = View.extend({
 
 const IframeView = View.extend({
   className: 'flex-grow',
-  template: hbs`<div class="js-dashboard-container"></div>`,
+  template: hbs``,
   ui: {
```

Comment:
there's no content for the `ui` here.  Also `template: false` should work here, but views without templates aren't "rendered" so you'd just change `onRender` to `initialize`